README.md

```diff
@@ -204,4 +206,13 @@ example file looks like this:
 
 responsive.metrics.api.key=ABCDEFGHIJKLMNOP
 responsive.metrics.secret=ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFG1234567890=
-```
\ No newline at end of file
+```
+
+### Getting `Exception opening socket` in the logs
+
+Make sure that you have added your local public IP address to the whitelist in the Storage tab
+on https://cloud.responsive.dev.
+
+### Helm install command failed with 403
+
+Make sure that your docker credentials are valid, or use `docker logout` to remove them altogether.
```

Comment:
we may want to specify `docker logout public.ecr.aws` since just `docker logout` didn't work for me